torch/_decomp/decompositions.py

```diff
@@ -19,6 +19,7 @@
     _safe_copy_out,
     out_wrapper,
 )
+from torch.ao.quantization.fx._decomposed import quantized_decomposed_lib  # noqa: F401
```

Comment:
I would suggest these decompositions for custom ops to live together with the operator definition. Something like ao/quantization/decompositions.py ?

User should be able to define custom op, define decompositions for custom op, all in user script.

Reply:
Let me know if this doesn't work... 
My goal is to enable dynamic decomposition registration for custom op.